npm-publish-branch-preview/entrypoint.sh

```diff
@@ -23,6 +23,6 @@ else
   echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc
   npm config set unsafe-perm true
   npm install
-  tag="$(echo $GITHUB_HEAD_REF | sed -E 's/\_+/&\_/g;s/\//\_/g')"
+  tag="$(echo $GITHUB_HEAD_REF | sed -E 's/\_/\_\_/g;s/\//\_/g')"
```

Comment:
I don't think that `_` is a special character in `sed` so shouldn't need to be escaped. The only thing that needs to be escaped is the `/` because it's used as the delimiter so 

`s/_/__/g;s/\//_/g`

ought to work. But since we're explicitly matching on `/` it might be better to use a different delimiter than the conventional `/` like `:`

`s:_:__:g;s:/:_:g;`

Reply:
Are the different delimiters just to make everything more legible or do they have different functionality?

Reply:
Nope, same functionality. Generally you use different delimiters when your matching directly involves the conventional delimiter `/`